src/Symfony/Bridge/Doctrine/Tests/DataCollector/DoctrineDataCollectorTest.php

```diff
@@ -84,6 +84,22 @@ public function testCollectQueries($param, $types, $expected, $explainable)
         $this->assertEquals($explainable, $collected_queries['default'][0]['explainable']);
     }
 
+    public function testCollectQueryWithNoParams()
+    {
+        $queries = array(
+            array('sql' => 'SELECT * FROM table1', 'params' => array(), 'types' => array(), 'executionMS' => 1),
+            array('sql' => 'SELECT * FROM table1', 'params' => null, 'types' => null, 'executionMS' => 1),
+        );
+        $c = $this->createCollector($queries);
+        $c->collect(new Request(), new Response());
+
+        $collected_queries = $c->getQueries();
```

Comment:
I just noticed it's also snaked_cased in the other test case. Feel free to fix that one as well.